ops/deploy.dev.sh

```diff
@@ -6,7 +6,7 @@ set -e
 
 # set any of these to "yes" to turn on watchers
 watch_ethprovider="no"
-watch_hub="yes"
+watch_hub="no"
```

Comment:
These are expensive (in terms of cpu usage & time to start-up) so let's keep them off by default & let devs selectively turn them on as needed.